src/apiary.apib

```diff
@@ -1392,6 +1392,29 @@ View all Creative for the specified Campaign.
     + Attributes
         + creatives (array[Creative Wrapper])
 
+## List all Available Feeds [GET /user/publisher/{publisher_id}/feed{?campaign_id,active}]
```

Comment:
This is the method for listing all feeds, can you also request an individual feed by its id? This is a recurring pattern in the docs, because it allows you to define the entity and have a specific dredd test against the entity attributes.